templates/config.edn

```diff
@@ -207,145 +242,179 @@
- ;; }
-
- ;; Mobile photo uploading setup
- ;; :mobile/photo
- ;; {:allow-editing? true
- ;;  :quality          80}
+ ;; Example usage:
+ ;; :ignored-page-references-keywords #{:author :website}
+ :ignored-page-references-keywords #{}
+
+ ;; logbook configuration.
+ :logbook/settings 
+ {; Disable logbook. (Default value: false)
+  :enabled-in-timestamped-blocks false
+  ; limit logbook to minutes, eliminating seconds. (Default value: false)
+  :with-second-support? false
+  ; display logbook in all blocks after time-tracking. (Default value: false)
+  :enabled-in-all-blocks false}
+
+ ;; Mobile photo upload configuration.
+ :mobile/photo
+ {:allow-editing? true
+  :quality        80}
 
  ;; Mobile features options
  ;; Gestures
- ;; :mobile
+ ;; Example usage:
+ ;; :mobile 
  ;; {:gestures/disabled-in-block-with-tags ["kanban"]}
+ :mobile
+ {:gestures/disabled-in-block-with-tags []}
 
  ;; Extra CodeMirror options
  ;; See https://codemirror.net/5/doc/manual.html#config for possible options
+ ;; Example usage:
  ;; :editor/extra-codemirror-options {:keyMap "emacs" :lineWrapping true}
+ :editor/extra-codemirror-options
+ {:lineWrapping  false  ; Default value: false
+  :lineNumbers   true   ; Default value: true
+  :readOnly      false} ; Default value: false
 
  ;; Enable logical outdenting
- ;; :editor/logical-outdenting? true
+ ;; default value: false
+  :editor/logical-outdenting? false
 
- ;; When both text and a file are in the clipboard, paste the file
- ;; :editor/preferred-pasting-file? true
+ ;; Prefer pasting the file when text and a file are in the clipboard.
+ ;; default value: false
+  :editor/preferred-pasting-file? false
 
- ;; Quick capture templates for receiving contents from other apps.
+ ;; Quick capture templates for receiving content from other apps.
  ;; Each template contains three elements {time}, {text} and {url}, which can be auto-expanded
- ;; by received contents from other apps. Note: the {} cannot be omitted.
+ ;; by receiving content from other apps. Note: the {} cannot be omitted.
  ;; - {time}: capture time
  ;; - {date}: capture date using current date format, use `[[{date}]]` to get a page reference
  ;; - {text}: text that users selected before sharing.
- ;; - {url}: url or assets path for media files stored in Logseq.
- ;; You can also reorder them, or even only use one or two of them in the template.
- ;; You can also insert or format any text in the template as shown in the following examples.
+ ;; - {url}: URL or assets path for media files stored in Logseq.
+ ;; You can also reorder them or use only one or two of them in the template.
+ ;; You can also insert or format any text in the template, as shown in the following examples.
  ;; :quick-capture-templates
- ;; {:text "[[quick capture]] **{time}**: {text} from {url}"
- ;;  :media "[[quick capture]] **{time}**: {url}"}
+  :quick-capture-templates
+  {:text "[[quick capture]] **{time}**: {text} — {url}"
+   :media "[[quick capture]] **{time}**: {url}"}
```

Comment:
@andelf is it ok to add an em dash (—) to the default text template to separate the text and URL? In the previously suggested template, the word 'from' was used instead. The default value does not contain a separator between the text and the URL. I strongly believe that adding an em dash (—) would improve the default template. It is a small change, but I think it's a good quality change.

Please let me know if I should revert it.

```cljs
        template (get-in (state/get-config)
                         [:quick-capture-templates :text]
                         "**{time}** [[quick capture]]: {text} — {url}")
```